### PR TITLE
Dev me18 examples

### PR DESCRIPTION
**Pull request by ozersa**
MAX32690 examples reviewed
examples table [here](https://confluence.maxim-ic.com/display/MSW/MSDK+Example+Validation+and+Adding+Missing+Boards)
The issue list [here](https://jira.maxim-ic.com/browse/MSDK-659?src=confmacro)

### CODE REVIEW

**Review by sihyung-maxim — DISMISSED** [review]

File: Examples/MAX32690/HBMC/main.c [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    mem.device_type = MXC_HPB_DEV_HYPER_RAM;
+    mem.cfg_reg_val = cfg_reg;
+    mem.cfg_reg_val_len = 1;
+    mem.read_cs_high = MXC_HPB_CS_HIGH_10_5;
+    mem.write_cs_high = MXC_HPB_CS_HIGH_10_5;
+    mem.read_cs_setup = MXC_HPB_CS_SETUP_HOLD_16;
+    mem.write_cs_setup = MXC_HPB_CS_SETUP_HOLD_14;
+    mem.read_cs_hold = MXC_HPB_CS_SETUP_HOLD_5;
+    mem.write_cs_hold = MXC_HPB_CS_SETUP_HOLD_12;
+    mem.latency_cycle = MXC_HPB_LATENCY_6;
```
Comment (sihyung-maxim) [inline Examples/MAX32690/HBMC/main.c:152]: Please update to MXC_HPB_LAT_6.

File: Examples/MAX32690/HBMC/main.c [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    mem.device_type = MXC_HPB_DEV_XCCELA_PSRAM;
+    mem.cfg_reg_val = cfg_reg;
+    mem.cfg_reg_val_len = 1;
+    mem.read_cs_high = MXC_HPB_CS_HIGH_2_5;
+    mem.write_cs_high = MXC_HPB_CS_HIGH_2_5;
+    mem.read_cs_setup = MXC_HPB_CS_SETUP_HOLD_2;
+    mem.write_cs_setup = MXC_HPB_CS_SETUP_HOLD_2;
+    mem.read_cs_hold = MXC_HPB_CS_SETUP_HOLD_2;
+    mem.write_cs_hold = MXC_HPB_CS_SETUP_HOLD_2;
+    mem.latency_cycle = MXC_HPB_LATENCY_5;
```
Comment (sihyung-maxim) [inline Examples/MAX32690/HBMC/main.c:112]: Please update to MXC_HPB_LAT_5.

**Review by sihyung-maxim — CHANGES_REQUESTED** [review]
There are still some leftover build errors. These two were the only ones I found.

File: Examples/MAX32690/HBMC/main.c (hunk 2 above)
Comment (ozersa) [reply Examples/MAX32690/HBMC/main.c]: Updated

**Review by sihyung-maxim — APPROVED** [review]

**Review by Jake-Carter — COMMENTED** [review]
Looks good.  One minor issue to resolve to get the HBMC project compatible with the SBT rules.

File: Examples/MAX32690/HBMC/project.mk [hunk 3]
```diff
@@ -8,3 +8,6 @@
 # **********************************************************
 
 # Add your config here!
+
+# Override the default linkerfile
+LINKERFILE=$(TARGET_LC)_hpb.ld
```
[the remark this answers is not in the document]
Comment (ozersa) [reply Examples/MAX32690/HBMC/project.mk]: Good catch!
Fix [here](https://github.com/Analog-Devices-MSDK/msdk/pull/169/commits/e6fb0103f356f1f6ae4477fb833b09b9fb8bdd7b)

Related but different topic:
This case demonstrate that: For current structure user can not overwrite linker file for SBT=1 mode.
Depend on your above build, the linker file set to max32690_hpb.ld , but due to SBT=1 the linker file force to use max32690_sla.ld.
Might need to be considered in future.

**Review by sihyung-maxim — APPROVED** [review]

File: Examples/MAX32690/HBMC/project.mk (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jake-Carter) [reply Examples/MAX32690/HBMC/project.mk]: True, we might want to move the default linkerfile selection to gcc.mk in the future.  Then SBT-config.mk can use `?=` to override the linkerfile only if none is specified by project.mk

**Review by Jake-Carter — APPROVED** [review]
Looks good!

**Comment by ozersa** [comment]
@sihyung-maxim , @Jake-Carter Could you rebase and merge it, I do not have permission to do it.
Thanks

**Comment by lorne-maxim** [comment]
Sadik, I have restricted the merges into main as squash and merge only.  I realize some history will be lost, but I want to keep the main branch's history as clean as possible.  Its commit history will be used to generate the release notes for each release.  If the main branch becomes cluttered with too many intermediate commits (e.g., all the "clang-format bot reformatting" messages) from each PR, creating the release notes is going to be challenging.  Because we are restricted to squash and merge only, I'd rather have a different PR for each feature or bug fix.  Alternatively, editing the commit notes of the squashed commit to include only the major points of that PR will also work.

**Comment by ozersa** [comment]
@lorne-maxim  that's OK. 
Could anyone merge it to the main branch?
I do not have permission to do it.
Thanks.

**Comment by lorne-maxim** [comment]
@ozersa, This PR has been merged.  You can delete the branch if you are finished with it.

**Pull request merged**